.gitignore

```diff
@@ -1,5 +1,6 @@
 # Build artifacts
 /book/
+/exercise-templates/
```

Comment:
Is this actually necessary? When I ran the code on the branch, it created `book/exerciser/` as expected.

Reply:
Not anymore, this was leftover from a previous iteration. Removed.